requirements.txt

```diff
@@ -1,40 +1,41 @@
 betterproto>=1.2.3
 click==7.1.2
 docker>=4.2.0
-fastapi<=0.61.2,>=0.58.0
+fastapi~=0.63.0
```

Comment:
Why do you change the following packages? Is the old version have some issue? Please explain.

Reply:
It is produced by my IDE automatically. I have cancelled these changes now.